app/liquid/liquid_renderer.rb

```diff
@@ -43,6 +48,14 @@ def images
 
   private
 
+  def member_data
+    {
+      url_params: @url_params,
+      member:     @member.try(:liquid_data),
+      location:   location
```

Comment:
You tell me... this hasn't changed. I suspect it's the method you want to reference rather that the ivar instantiated on initialisation.